modules/ui/bin/jenkins-build-deploy.sh

```diff
@@ -23,6 +23,8 @@
 #   nexus_repository_id          : nexus milestone repository id
 #   nexus_snapshot_repository_id : nexus snapshot repository id
 
+#This file assumes that this script is being executed from the root directory of project. E.g. .../wasabi/
```

Comment:
@iizrailevsky @abhishekjain88 This update is NOT specific to the atomic experiment creation feature.. During testing I found creating UI zip was failing and so I had to fix that in  jenkins-build-deploy.sh file... Root cause: not able to retrieve project.version properly...